modules/loadbalancer/main.tf

```diff
@@ -11,6 +12,72 @@ resource "aws_lb" "this" {
 
   enable_deletion_protection = false
   drop_invalid_header_fields = true
+
+  access_logs {
+    bucket  = aws_s3_bucket.lb-logs.bucket
+    prefix  = "${var.solution_name}-alb"
+    enabled = true
+  }
+}
+
+#tfsec:ignore:aws-s3-specify-public-access-block
```

Comment:
Only allowed are the following ignores in our case. All other tfsec issues are valid and should be tackled. Look at our S3 module to get some ideas.

# tfsec:ignore:aws-s3-enable-bucket-logging tfsec:ignore:aws-s3-enable-versioning tfsec:ignore:aws-s3-encryption-customer-key

Reply:
During the last git commit, it showed this all tfsec:ignores, but now it worked out with these 3 tfsec:ignores.